runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -106,6 +112,104 @@ export default class ExcelActionHandler extends StatefulActionHandler {
+                input.macro,
+                input.functionParams[0],
+                input.functionParams[1],
+                input.functionParams[2],
+                input.functionParams[3],
+            );
+        } else if (input.functionParams.length === 5) {
+            return this.session.Run(
+                input.macro,
+                input.functionParams[0],
+                input.functionParams[1],
+                input.functionParams[2],
+                input.functionParams[3],
+                input.functionParams[4],
+            );
+        } else if (input.functionParams.length === 6) {
+            return this.session.Run(
+                input.macro,
+                input.functionParams[0],
+                input.functionParams[1],
+                input.functionParams[2],
+                input.functionParams[3],
+                input.functionParams[4],
+                input.functionParams[5],
+            );
+        } else if (input.functionParams.length === 7) {
+            return this.session.Run(
+                input.macro,
+                input.functionParams[0],
+                input.functionParams[1],
+                input.functionParams[2],
+                input.functionParams[3],
+                input.functionParams[4],
+                input.functionParams[5],
+                input.functionParams[6],
+            );
+        } else if (input.functionParams.length === 8) {
+            return this.session.Run(
+                input.macro,
+                input.functionParams[0],
+                input.functionParams[1],
+                input.functionParams[2],
+                input.functionParams[3],
+                input.functionParams[4],
+                input.functionParams[5],
+                input.functionParams[6],
+                input.functionParams[7],
+            );
+        } else if (input.functionParams.length === 9) {
+            return this.session.Run(
+                input.macro,
+                input.functionParams[0],
+                input.functionParams[1],
+                input.functionParams[2],
+                input.functionParams[3],
+                input.functionParams[4],
+                input.functionParams[5],
+                input.functionParams[6],
+                input.functionParams[7],
+                input.functionParams[8],
+            );
+        } else if (input.functionParams.length === 10) {
+            return this.session.Run(
+                input.macro,
+                input.functionParams[0],
+                input.functionParams[1],
+                input.functionParams[2],
+                input.functionParams[3],
+                input.functionParams[4],
+                input.functionParams[5],
+                input.functionParams[6],
+                input.functionParams[7],
+                input.functionParams[8],
+                input.functionParams[9],
+            );
+        }
+    }
```

Comment:
i must say I'm impressed

Reply:
let's do it again on April 1st but this time i'll give you instant approve

Reply:
Unfortunately is's not a joke XD

Reply:
the Run command does not support destructuring becouse it doesn't have join function

Reply:
![image](https://github.com/runbotics/runbotics/assets/44839807/1ebcf4ea-33e7-4e52-ad31-196b657d74af)

Reply:
sorry @Stanlee77, I would more than happy to tell you its a joke but we are operating on a Visual Basic level and we cannot do better

Reply:
It would be helpful to add a tooltip with information that max 10 parameters are allowed and maybe a blocker after adding 10 parameters (disable button "+ ADD ITEM" once there are 10 params)

Reply:
There is already limit on frontend. But I also added condition on backend.